packages/admin-stories/src/docs/comet/admin/savesplitbutton.stories.mdx

```diff
@@ -0,0 +1,47 @@
+import { Meta, Story, Canvas } from "@storybook/addon-docs/blocks";
+
+import { SplitButton, SaveButton } from "@comet/admin";
+import { FormattedMessage } from "react-intl";
+
+# Components
+
+<Meta title="Docs/@comet/admin/savesplitbutton" />
+
+## Save / Save and go Back Split Button
+
```

Comment:
would be nice to have a short text describing the use case when this component can be used

Reply:
yeah of course, i will do that as soon as we can say the API from SplitButton and SaveButton is final